components/config/src/config/markup.rs

```diff
@@ -234,6 +240,7 @@ impl Default for Markdown {
             external_links_target_blank: false,
             external_links_no_follow: false,
             external_links_no_referrer: false,
+            external_links_external: false,
```

Comment:
I think it's fine to default to true. It's not going to break anything

Reply:
Fantastic :)